pkg/uroot/uroot.go

```diff
@@ -255,7 +255,9 @@ func CreateInitramfs(logger logger.Logger, opts Opts) error {
 			if err != nil {
 				return err
 			}
-			if err := archive.AddRecord(cpio.Symlink("init", rtarget)); err != nil {
+			if rtarget == "init" {
+				logger.Printf("not creating symlink init -> init")
```

Comment:
Capitalize Not, and say why

Not creating symlink init -> init, most likely because init already exists

Reply:
will this be another warning people ignore to their later surprise? Should it be an error with some explanation? 
Sorry for my confusion on this one.

Reply:
OK, how about this:
return fmt.Errorf("You are replacing u-root /init with your own. If this is your intent, you must also specify -initcmd=\"\" on the command line")

I realize it's a bit verbose but the consequences of getting this wrong, especially in firmware, are pretty dire.

Reply:
@mpictor I'd be good with this